run-dev.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cp -f ~/work/lndr/lndr-backend/data/lndr-server.dev.config ~/work/lndr/lndr-backend/data/lndr-server.config
+
+lndr-server
```

Comment:
Should probably be `./lndr-server` -- otherwise this will look only in the `PATH` to resolve, which may not be the version you want.